### ISSUE TITLE
iOS14 to iOS15/iOS16 with shsh2

### ISSUE THREAD

**Issue by piaomiaoguhong10** [m1stadev/deleo#14]
See, bro, My A10x iPad Pro is on iPadOS 14.3 and have shsh2 of iPadOS15/16, now I wanna use this tool to go to iPadOS15/16. Could I make it?

**Comment by m1stadev** [comment]
15.x restores should be possible, 16.x restores I believe will only work with 16.6 and later.

**Comment by zhoushiqi-flow** [comment]
Does this tool upgrade from ios14 to ios15 first upgrade the device to the latest and then downgrade it to ios15?

**Comment by m1stadev** [comment]
> Does this tool upgrade from ios14 to ios15 first upgrade the device to the latest and then downgrade it to ios15?

No, it only does one restore to the target version, but allows for flashing a SEP(/baseband) firmware from a different firmware (must still be signed), which allows for up/downgrades to non-latest firmwares in some cases.
